backends/build_system/utils.py

```diff
@@ -121,7 +121,16 @@ def _parse_req_line(line: str):
 
 
 def get_install_requires():
-    import flit_core.buildapi
+    try:
+        import flit_core.buildapi
+    except ImportError:
+        flit_core_range = get_flit_core_range()
+        raise RuntimeError(
+            f'In order to check dependencies the environment must have '
```

Comment:
I wonder if it makes sense to make mention of the `--with-download-deps` flag here as well?